site/docs/v1/tech/installation-guide/docker.adoc

```diff
@@ -10,102 +10,114 @@ navcategory: installation
-Review the link:/docs/v1/tech/reference/configuration[Configuration - Environment Variables] documentation to customize your deployment.
-
 == Docker Services
-In the above example configurations you will find a database, search and FusionAuth service. Read below to better understand how each service is configured.
+
+In the above example configurations there are three services:
+
+* database 
+* search 
+* FusionAuth
+
+Let's look at each of these.
 
 === Database Service
 
-At a minimum, you will need to either set the `POSTGRES_PASSWORD` environment variable in the `db` service section, or more ideally set the value in the host environment and leave it out of the `docker-compose.yml` file. Ensure the other properties fit your requirements. Refer to the link:/docs/v1/tech/installation-guide/system-requirements[System Requirements] for database version support.
+The database service provides a postgreSQL database for use by FusionAuth.
+
+You will need to either set the `POSTGRES_PASSWORD` environment variable in the `db` service section, or more ideally, set the value in the host environment and leave it out of the `docker-compose.yml` file.
+
+By default this database is not accessible outside of the docker compose containers, but you may expose the port if you want to examine the database.
+
+Review the other properties to ensure they meet your requirements. 
 
 === Search Service
 
-We currently support Elasticsearch versions 6.3.x - 7.6.x.  Later versions may work as well, but may not have been tested for compatibility.  Please let us know if you have a requirement for a different version of Elasticsearch. The remainder of the properties can be changed to whatever you need.
+include::docs/v1/tech/installation-guide/_elastic-search-version.adoc[]
 
-== Production Deployment
+include::docs/v1/tech/installation-guide/_elasticsearch-ram.adoc[]
 
-Elasticsearch has a few runtime requirements that may not be met by default on your host platform. Please review the Elasticsearch Docker production mode guide for more information.
+=== The FusionAuth Service
 
-* https://www.elastic.co/guide/en/elasticsearch/reference/7.6/docker.html#docker-cli-run-prod-mode
+This is the service that runs the FusionAuth application.
 
-For example if startup is failing and you see the following in the logs, you will need to increase `vm.max_map_count` on your host VM.
+Review the link:/docs/v1/tech/reference/configuration#options[Configuration] documentation to customize your deployment. The best way to configure FusionAuth when using Docker is to use environment variables as documented in that link.
+
+In addition, the port `9011` is mapped between the container and your host machine. This may be modified if you want to run more than one FusionAuth instance or for any other reason.
+
+=== Other Services
+
+You may add other services to the docker compose files if needed.
+
+For example, in development environments it can be helpful to run mailcatcher for SMTP.
```

Comment:
MailCatcher has capitalization in its name. Should we provide a link to the site?

Reply:
Good catch